src/status_im/utils/number.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im.utils.number
+  "Contains helpers for numbers"
+  (:require [clojure.string :refer [upper-case]]))
+
+(defn- number-to-nk [number-string capitalize?]
+  (let [numeric-value (if (> 0 number-string)
+                        (throw (str "Invalid input " number-string))
+                        (js/parseInt number-string))]
+    (if-not (js/isNaN numeric-value)
+      (if (<= 1000 numeric-value)
+        (str (quot numeric-value 1000) (if capitalize?
+                                         (upper-case "k")
+                                         "k"))
+        (str numeric-value))
+      "Invalid Number")))
+
+(defn number->nk
+  "Returns the thousands in a number in NK format
```

Comment:
consider adding some examples like this:
Examples
1000 -> 1K
10000 -> 10K
10001 -> 10K
10100 -> 10.1K

I didn't know what NK was until I checked the issue description

Reply:
Thanks, Improved